open-api/rest-catalog-open-api.yaml

```diff
@@ -2106,6 +2208,139 @@ components:
           items:
             $ref: '#/components/schemas/PartitionStatisticsFile'
 
+    PlanTask:
+      description:
+        A JSON object that contains information provided by the server,
+        to be utilized by clients for distributed planning, should be supplied
+        as is for input in PlanTable operation.
+      type: object
+
+    FileScanTask:
+      type: object
+      required:
+        - schema
+        - spec
+        - start
+        - length
+        - data-file
+      properties:
+        data-file:
+          $ref: '#/components/schemas/ContentFile'
+        start:
+          type: number
+        length:
+          type: number
+        delete-files:
+          type: array
+          items:
+            $ref: '#/components/schemas/ContentFile'
+        schema:
+          description: Table Schema
+          $ref: '#/components/schemas/Schema'
+        spec:
+          $ref: '#/components/schemas/PartitionSpec'
+        residual-filter:
+          description: An optional residual filter provided by a service, if not present clients shall calculate this residual or use the original filter.
+          $ref: '#/components/schemas/Expression'
+
+    TypeValue:
```

Comment:
looks like these models are not fully removed after the rebase?

Reply:
Will remove this.